ROMFS/px4fmu_common/init.d/rc.uuv_apps

```diff
@@ -0,0 +1,26 @@
+#!/bin/sh
+#
+# Standard apps for uuvs. Attitude/Position estimator, Attitude/Position control.
+#
+# NOTE: Script variables are declared/initialized/unset in the rcS script.
+#
+
+###############################################################################
+#                       Begin Estimator Group Selection                       #
+###############################################################################
+
+ekf2 start
+
+###############################################################################
+#                        End Estimator Group Selection                        #
+###############################################################################
+
+#
+# Start UUV Attitude Controller.
+#
+uuv_att_control start
+
+#
+# Start UUV Land Detector.
+#
+land_detector start rover
```

Comment:
Sounds like we need another vehicle type.

Reply:
agree!
afaik rover landdetector is basically just a dummy, right?
what do you recommend?